### ISSUE TITLE
Indexing Hangs on v0.1.0

### ISSUE THREAD

**Issue by ryan-clancy** [osirrc/pisa-docker#15]
Using `v0.1.0`, indexing hangs after printing out lots of messages like:
```
[2019-06-17 21:51:45.668] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADZFT</PROFILE>
[2019-06-17 21:51:45.668] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADYFT</PROFILE>
[2019-06-17 21:51:45.668] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADXFT</PROFILE>
[2019-06-17 21:51:45.668] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADWFT</PROFILE>
[2019-06-17 21:51:45.668] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADVFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADUFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADTFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADSFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADRFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADQFT</PROFILE>
[2019-06-17 21:51:45.669] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADPFT</PROFILE>
[2019-06-17 21:51:45.670] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADOFT</PROFILE>
[2019-06-17 21:51:45.670] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADNFT</PROFILE>
[2019-06-17 21:51:45.670] [warning] Skipped invalid record: Could not consume any tag  in context: PROFILE>_AN-EA1CHADMFT</PROFILE>
```

Using `v0.0.1` it seems to work properly. Are there changes in `v0.1.0` that might have broken indexing?

**Comment by amallia** [comment]
It should be fixed with v0.1.1.
Sorry but I pushed some crap while pushing the v0.1.0.

Feel free to reopen if it is not fixed.

**Comment by ryan-clancy** [comment]
I get really low MAP and P.30 with `v0.1.1`:
```
num_q                   all     149
map                     all     0.0002
P_30                    all     0.0009
```

`prepare` log: https://drive.google.com/file/d/1yeqNraJQEPTVtQOfwkHpkTiffRegOA6k/view?usp=drivesdk
`search` log: https://drive.google.com/file/d/19ffJgThztA9_nmVMZbCxvJL874QGZSxP/view?usp=drivesdk
run file: https://drive.google.com/file/d/1EyW3JUJbfxO3y7oFH5ZJi6qiGQVJ7SgX/view?usp=drivesdk

**Comment by JMMackenzie** [comment]
Ryan,

It looks like an issue with the parsing of the query file (I think) as the run file has entries like this:

```
0	Q0	FR940630-0-00035	0	4.3528	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FBIS4-30564	1	4.33725	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FBIS4-30519	2	4.33717	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FBIS3-40101	3	4.30415	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FBIS4-30521	4	4.30203	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FT942-5682	5	4.29526	block_max_wand-block_simdbp-vbmw-porter2
0	Q0	FR940304-1-00160	6	4.28849	block_max_wand-block_simdbp-vbmw-porter2
```

and

```
<num> Number	Q0	FR941216-2-00090	0	8.78309	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940822-2-00057	1	8.77518	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941230-0-00013	2	8.71654	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940907-0-00044	3	8.68593	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941202-1-00043	4	8.6352	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940607-0-00013	5	8.51706	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941230-0-00012	6	8.49256	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940913-0-00011	7	8.48086	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940118-2-00044	8	8.47459	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FT944-17926	9	8.45973	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940725-2-00054	10	8.44187	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FBIS3-5261	11	8.43981	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941202-1-00044	12	8.41506	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941202-1-00045	13	8.39659	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR941230-0-00031	14	8.39301	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940118-0-00044	15	8.38319	block_max_wand-block_simdbp-vbmw-porter2
<num> Number	Q0	FR940325-0-00008	16	8.38254	block_max_wand-block_simdbp-vbmw-porter2
```

Could you show us how you called the `search` binary so we can try to reproduce?

**Comment by amallia** [comment]
Problem should be solved with `v0.1.2`
I introduced a breaking change with `v0.1.1` which is gone now.

Feel free to reopen
